packages/@sanity/form-builder/src/inputs/BlockEditor/BlockEditor.js

```diff
@@ -226,6 +229,14 @@ export default class BlockEditor extends React.PureComponent<Props, State> {
     return null
   }
 
+  handleOnDragEnter = (event: SyntheticDragEvent<>) => {
+    this.setState({isDragging: true})
+  }
+
+  handleOnDragLeave = debounce((event: SyntheticDragEvent<>) => {
```

Comment:
It would be good to cancel the debounce on componentWillUnmount here. I think it should be a matter of adding this
```js
componentWillUnmount() {
  this.handleOnDragEnter.cancel()
}
```

Reply:
Good catch, fixed.